source/develop/release-management/process/making-an-release.md

```diff
@@ -2,24 +2,168 @@
+
+```bash
+# CentOS Stream 8
+cbs tag virt8s-ovirt-45-release <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt8s-ovirt-45-release ovirt-hosted-engine-ha-2.4.10-1.el8
+```
+
+this will publish the build on http://mirror.centos.org/centos/8-stream/virt/ repository and to all CentOS mirrors.
+
+
+```bash
+# CentOS Stream 9
+cbs tag virt9s-ovirt-45-release <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt9s-ovirt-45-release ovirt-hosted-engine-ha-2.4.10-1.el9
+```
+
+this will publish the build on http://mirror.stream.centos.org/SIGs/9-stream/virt/ repository and to all CentOS mirrors.
+
+## Release notes preparation
+
+Release notes can be autogenerated from git logs.
+Please follow instructions on [oVirt Release Engineering Tools documentation](https://github.com/oVirt/releng-tools/blob/master/releases/README-prepare-patches).
+
+Once ready, the notes can be automatically generated with [`release_notes_git.py`](https://github.com/oVirt/releng-tools/blob/master/release_notes_git.py) command line like:
+
+```bash
+# this assumes milestone/ovirt-4.5.0.conf being already populated
+# https://github.com/oVirt/releng-tools/blob/master/milestones/ovirt-4.5.0.conf
+ ./release_notes_git.py --git-basedir /var/tmp/release-cache --contrib-project-list ovirt-4.5.0 >notes.md
+```
+
+## Announcements
+
+Once ready to announce please send the announcement:
+
+To [oVirt Blog](https://blogs.ovirt.org/)
+
+To the relevant mailing list:
+
+* [Announce](https://lists.ovirt.org/archives/list/announce@ovirt.org/)
+* [Users](https://lists.ovirt.org/archives/list/users@ovirt.org/)
+* [Devel](https://lists.ovirt.org/archives/list/devel@ovirt.org/)
+
+To social media:
+
+* [Twitter](https://twitter.com/ovirt)
+* [Reddit](https://www.reddit.com/r/ovirt)
+* [Linkedin Official](https://www.linkedin.com/company/ovirt)
+* [Linkedin oVirt group](https://www.linkedin.com/groups/4707460/)
+* [Facebook Official](https://www.facebook.com/groups/ovirt.openvirtualization)
+
+To local communities:
+
+* [Linkedin oVirt Italia](https://www.linkedin.com/groups/13669751/)
+* [Facebook oVirt India](https://www.facebook.com/groups/409421802961475/)
+* [Facebook oVirt Italia](https://www.facebook.com/groups/ovirt.italia/)
+* [Facebook oVirt Korea](https://www.facebook.com/groups/ovirt.korea/)
+* [Facebook oVirt Malaysia](https://www.facebook.com/groups/ovirtUGMY/)
+* [Facebook oVirt Myanmar](https://www.facebook.com/Ovirt-Myanmar-Community-974969229309990)
+* [Facebook oVirt Philippines](https://www.facebook.com/groups/ovirtph/)
 
-This page should ideally have a checklist of things we need to do before a release, including:
+Missed someone? Please add!
 
-*   Release notes
-*   Release announcement
-*   Communication [press plan](/develop/release-management/process/press-plan.html)
-*   Release talking points which can be used as the basis for review articles
-*   Collected links related to the release
+Typical announce for social media:
```

Comment:
A typical announcement for social media:

Reply:
ack